src/workflow/StreamlitUI.py

```diff
@@ -100,14 +126,23 @@ def upload_widget(
 
         # Local file upload option: via directory path
         if st.session_state.location == "local":
-            c2_text, c2_checkbox = c2.columns([2, 1])
+            c2_text, c2_checkbox = c2.columns([1.5, 1], gap="large")
             c2_text.markdown("**OR copy files from local folder**")
-            use_symlink = c2_checkbox.checkbox("Use symlinks", key=f"{key}-symlink", help="Use symbolic links instead of copying files.")
-            with c2.form(f"{key}-local-file-upload"):
-                local_dir = st.text_input(f"path to folder with **{name}** files")
-                if st.form_submit_button(
-                    f"Copy **{name}** files from local folder", use_container_width=True
-                ):
+            use_copy = c2_checkbox.checkbox("Make a copy of files", key=f"{key}-copy_files", value=True, help="Create a copy of files in workspace.")
```

Comment:
Changed the use symlinks button to make a copy of files button, which is now by default set to true, so that copies are made in the web app workspace directory.